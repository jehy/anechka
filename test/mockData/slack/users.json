{
  "ok": true,
  "members": [
    {
      "id": "U02C2K9UR",
      "team_id": "T025RJ98W",
      "name": "konstantin.kalinin",
      "deleted": false,
      "color": "8f4a2b",
      "real_name": "Konstantin Kalinin",
      "tz": "Europe/Moscow",
      "tz_label": "Moscow Time",
      "tz_offset": 10800,
      "profile": {
        "title": "",
        "phone": "",
        "skype": "",
        "real_name": "Konstantin Kalinin",
        "real_name_normalized": "Konstantin Kalinin",
        "display_name": "konstantin.kalinin",
        "display_name_normalized": "konstantin.kalinin",
        "status_text": "",
        "status_emoji": "",
        "status_expiration": 0,
        "avatar_hash": "fc9943306458",
        "image_original": "https://avatars.slack-edge.com/2016-11-08/101029133712_fc99433064583483f509_original.jpg",
        "email": "konstantin.kalinin@domain.com",
        "first_name": "Konstantin",
        "last_name": "Kalinin",
        "image_24": "https://avatars.slack-edge.com/2016-11-08/101029133712_fc99433064583483f509_24.jpg",
        "image_32": "https://avatars.slack-edge.com/2016-11-08/101029133712_fc99433064583483f509_32.jpg",
        "image_48": "https://avatars.slack-edge.com/2016-11-08/101029133712_fc99433064583483f509_48.jpg",
        "image_72": "https://avatars.slack-edge.com/2016-11-08/101029133712_fc99433064583483f509_72.jpg",
        "image_192": "https://avatars.slack-edge.com/2016-11-08/101029133712_fc99433064583483f509_192.jpg",
        "image_512": "https://avatars.slack-edge.com/2016-11-08/101029133712_fc99433064583483f509_512.jpg",
        "image_1024": "https://avatars.slack-edge.com/2016-11-08/101029133712_fc99433064583483f509_1024.jpg",
        "status_text_canonical": "",
        "team": "T025RJ98W",
        "is_custom_image": true
      },
      "is_admin": false,
      "is_owner": false,
      "is_primary_owner": false,
      "is_restricted": false,
      "is_ultra_restricted": false,
      "is_bot": false,
      "is_app_user": false,
      "updated": 1557839190
    },
    {
      "id": "U02K307KL",
      "team_id": "T025RJ98W",
      "name": "pavel.vlasov",
      "deleted": false,
      "color": "674b1b",
      "real_name": "Pavel Vlasov",
      "tz": "Europe/Moscow",
      "tz_label": "Moscow Time",
      "tz_offset": 10800,
      "profile": {
        "title": "whoami",
        "phone": "",
        "skype": "enotvl",
        "real_name": "Pavel Vlasov",
        "real_name_normalized": "Pavel Vlasov",
        "display_name": "pavel.vlasov",
        "display_name_normalized": "pavel.vlasov",
        "status_text": "",
        "status_emoji": ":kiddingme:",
        "status_expiration": 0,
        "avatar_hash": "2c4eaf6b8b49",
        "image_original": "https://avatars.slack-edge.com/2016-11-09/101677120752_2c4eaf6b8b49962c48b4_original.jpg",
        "email": "pavel.vlasov@domain.com",
        "first_name": "Pavel",
        "last_name": "Vlasov",
        "image_24": "https://avatars.slack-edge.com/2016-11-09/101677120752_2c4eaf6b8b49962c48b4_24.jpg",
        "image_32": "https://avatars.slack-edge.com/2016-11-09/101677120752_2c4eaf6b8b49962c48b4_32.jpg",
        "image_48": "https://avatars.slack-edge.com/2016-11-09/101677120752_2c4eaf6b8b49962c48b4_48.jpg",
        "image_72": "https://avatars.slack-edge.com/2016-11-09/101677120752_2c4eaf6b8b49962c48b4_72.jpg",
        "image_192": "https://avatars.slack-edge.com/2016-11-09/101677120752_2c4eaf6b8b49962c48b4_192.jpg",
        "image_512": "https://avatars.slack-edge.com/2016-11-09/101677120752_2c4eaf6b8b49962c48b4_512.jpg",
        "image_1024": "https://avatars.slack-edge.com/2016-11-09/101677120752_2c4eaf6b8b49962c48b4_512.jpg",
        "status_text_canonical": "",
        "team": "T025RJ98W",
        "is_custom_image": true
      },
      "is_admin": false,
      "is_owner": false,
      "is_primary_owner": false,
      "is_restricted": false,
      "is_ultra_restricted": false,
      "is_bot": false,
      "is_app_user": false,
      "updated": 1537859553
    },
    {
      "id": "U18LQV66M",
      "team_id": "T025RJ98W",
      "name": "vadim.shilov",
      "deleted": false,
      "color": "385a86",
      "real_name": "Vadim Shilov",
      "tz": "Europe/Moscow",
      "tz_label": "Moscow Time",
      "tz_offset": 10800,
      "profile": {
        "title": "Back-end developer",
        "phone": "+7 (920) 173-72-09",
        "skype": "vadik.shilov",
        "real_name": "Vadim Shilov",
        "real_name_normalized": "Vadim Shilov",
        "display_name": "vadim.shilov",
        "display_name_normalized": "vadim.shilov",
        "status_text": "",
        "status_emoji": ":coffee:",
        "status_expiration": 0,
        "avatar_hash": "1e6c60ec7ec4",
        "image_original": "https://avatars.slack-edge.com/2018-06-22/386515574034_1e6c60ec7ec4393f6524_original.png",
        "email": "vadim.shilov@domain.com",
        "first_name": "Vadim",
        "last_name": "Shilov",
        "image_24": "https://avatars.slack-edge.com/2018-06-22/386515574034_1e6c60ec7ec4393f6524_24.png",
        "image_32": "https://avatars.slack-edge.com/2018-06-22/386515574034_1e6c60ec7ec4393f6524_32.png",
        "image_48": "https://avatars.slack-edge.com/2018-06-22/386515574034_1e6c60ec7ec4393f6524_48.png",
        "image_72": "https://avatars.slack-edge.com/2018-06-22/386515574034_1e6c60ec7ec4393f6524_72.png",
        "image_192": "https://avatars.slack-edge.com/2018-06-22/386515574034_1e6c60ec7ec4393f6524_192.png",
        "image_512": "https://avatars.slack-edge.com/2018-06-22/386515574034_1e6c60ec7ec4393f6524_512.png",
        "image_1024": "https://avatars.slack-edge.com/2018-06-22/386515574034_1e6c60ec7ec4393f6524_1024.png",
        "status_text_canonical": "",
        "team": "T025RJ98W",
        "is_custom_image": true
      },
      "is_admin": false,
      "is_owner": false,
      "is_primary_owner": false,
      "is_restricted": false,
      "is_ultra_restricted": false,
      "is_bot": false,
      "is_app_user": false,
      "updated": 1547280605
    },
    {
      "id": "U6DGBPPL6",
      "team_id": "T025RJ98W",
      "name": "evgeny.bondarenko",
      "deleted": false,
      "color": "53b759",
      "real_name": "Evgeny Bondarenko",
      "tz": "Europe/Moscow",
      "tz_label": "Moscow Time",
      "tz_offset": 10800,
      "profile": {
        "title": "avia backend",
        "phone": "9167964985",
        "skype": "jehy_rus",
        "real_name": "Evgeny Bondarenko",
        "real_name_normalized": "Evgeny Bondarenko",
        "display_name": "evgeny.bondarenko",
        "display_name_normalized": "evgeny.bondarenko",
        "status_text": "",
        "status_emoji": "",
        "status_expiration": 0,
        "avatar_hash": "7d283f8e2b6f",
        "image_original": "https://avatars.slack-edge.com/2018-10-11/453244450832_7d283f8e2b6f1c3e67c9_original.png",
        "email": "evgeny.bondarenko@domain.com",
        "first_name": "Evgeny",
        "last_name": "Bondarenko",
        "image_24": "https://avatars.slack-edge.com/2018-10-11/453244450832_7d283f8e2b6f1c3e67c9_24.png",
        "image_32": "https://avatars.slack-edge.com/2018-10-11/453244450832_7d283f8e2b6f1c3e67c9_32.png",
        "image_48": "https://avatars.slack-edge.com/2018-10-11/453244450832_7d283f8e2b6f1c3e67c9_48.png",
        "image_72": "https://avatars.slack-edge.com/2018-10-11/453244450832_7d283f8e2b6f1c3e67c9_72.png",
        "image_192": "https://avatars.slack-edge.com/2018-10-11/453244450832_7d283f8e2b6f1c3e67c9_192.png",
        "image_512": "https://avatars.slack-edge.com/2018-10-11/453244450832_7d283f8e2b6f1c3e67c9_512.png",
        "image_1024": "https://avatars.slack-edge.com/2018-10-11/453244450832_7d283f8e2b6f1c3e67c9_1024.png",
        "status_text_canonical": "",
        "team": "T025RJ98W",
        "is_custom_image": true
      },
      "is_admin": false,
      "is_owner": false,
      "is_primary_owner": false,
      "is_restricted": false,
      "is_ultra_restricted": false,
      "is_bot": false,
      "is_app_user": false,
      "updated": 1555705850
    }
  ],
  "cache_ts": 1557844444
}
