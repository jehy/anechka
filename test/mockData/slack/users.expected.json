{
  "konstantin.kalinin": "U02C2K9UR",
  "pavel.vlasov": "U02K307KL",
  "vadim.shilov": "U18LQV66M",
  "evgeny.bondarenko": "U6DGBPPL6",
  "lastUpdate": "2019-05-14T14:46:06.538Z"
}
